### PR TITLE
patientInfoForm interface reflects FMP

### PR DESCRIPTION
**Pull request by bmitchinson**
### Description: 
- Updates patientInfoForm to reflect FMP

### Notes:
na

### Screenshots (Optional): 
na

### Checklist:
- [x] 90% Test Coverage

### CODE REVIEW

File: pwa/src/components/PatientInfoForm.tsx [hunk 1]
```diff
@@ -109,37 +109,38 @@ export default function Form(): ReactElement {
...
-                complaintField1.value,
-                complaintField2.value,
-                complaintField3.value,
-            ],
-            history: history.value,
-            exam: exam.value,
+            albendazole_given: albendazole,
+            known_med_allergies: allergies.value,
+            meds_currently_taken: medicines.value,
+            chief_complaint: `1.) ${complaintField1.value} 2.) ${complaintField2.value} 3.) ${complaintField3.value}`,
```
Comment (abpwrs) [inline pwa/src/components/PatientInfoForm.tsx:141]: comma-separated values may be cleaner to parse when trying to retrieve this data from the database

File: pwa/src/state/models/patientInfoForm.model.ts [hunk 2]
```diff
@@ -2,33 +2,35 @@ import { Action, action } from 'easy-peasy'
...
+    ob_abortus?: number
+    ob_last_menstrual_period?: Date
+    ob_wants_planning?: YN
+    weight_in_lbs?: number
+    height_in_inches?: number
+    bp_systolic_blood_pressure?: number
+    bp_diastolic_blood_pressure?: number
+    temp_in_fahrenheit?: number
+    pulse_rate?: number
+    // TODO: Can't find zScore proper name?
```
Comment (abpwrs) [inline pwa/src/state/models/patientInfoForm.model.ts:26]: Is there a spike/issue for this so we don't loose track of it

Comment (bmitchinson) [reply pwa/src/state/models/patientInfoForm.model.ts:26]: There is now: #105

File: pwa/src/components/PatientInfoForm.tsx (hunk 1 above)
Comment (bmitchinson) [reply pwa/src/components/PatientInfoForm.tsx:141]: I disagree. What if a complaint has a comma included in it. Could we leave it for now and optimize later.

**Comment by bmitchinson** [comment]
> Should this still be stored correctly w/ s`easy-peasy`? I am unable to see the fields get updated

Do you mean that when you visit the route with the form, the contents don't reflect what's in easy peasy? There's a ticket for that #75

File: pwa/src/components/PatientInfoForm.tsx (hunk 1 above)
Comment (abpwrs) [reply pwa/src/components/PatientInfoForm.tsx:141]: Yup, we can optimize later.

Just to address the comma problem though, we could escape it when we generate the field, either way, we would need some pattern matching logic to parse this field.

**Comment by abpwrs** [comment]
> > Should this still be stored correctly w/ s`easy-peasy`? I am unable to see the fields get updated
> 
> Do you mean that when you visit the route with the form, the contents don't reflect what's in easy peasy? There's a ticket for that #75

When I fill out and submit the form on `/patientinfo` and then navigate to `/easy-peasy` I do not see the state information being rendered. Are you able to replicate this or is it a bug in my build?

**Comment by bmitchinson** [comment]
@abpwrs Looks like it's you? 

if you try appending a `--build` to rebuild your containers that might update things? idk. If you do that your database important will be safe, it's cached across container rebuilds i tried it.

Here's a screenshot of me on this branch, filling out the form with partial values, clicking submit, and seeing them `/easy-peasy`

image
image

**Comment by abpwrs** [comment]
Gotcha, thank you for checking that @bmitchinson

**Pull request merged**